GVFS/GVFS.Platform.Windows/WindowsFileSystemVirtualizer.cs

```diff
@@ -104,6 +104,11 @@ public static FSResult HResultToFSResult(HResult result)
                 case HResult.VirtualizationInvalidOp:
                     return FSResult.VirtualizationInvalidOperation;
 
+                // Experimental: This unexpected error code is returned by DeleteFile on Win server 2016
+                // while deleting a partial file. On Win 10, the same call returns VirtualizationInvalidOp.
+                case (HResult)(-2147024579):

```

Comment:
We will want to add this the to a constant somewhere so it has a good name.  Also since this is the same as `HResult.VirtualizationInvalidOp` we can just add this case above it like:
```
case <our const>:
case HResult.VirtualizationInvalidOp:
```

Reply:
Updated... take a look.